web/src/upload.js

```diff
@@ -446,6 +453,11 @@ export function initialize() {
     $(".app, #navbar-fixed-container").on("drop", (event) => {
         event.preventDefault();
 
+        if (event.target.nodeName === "IMG" && event.target === drag_drop_img) {
+            drag_drop_img = null;
+            return;
```

Comment:
If you're setting it to `null` to clear it here, you should also have it initialized to `null` above. It doesn't make sense to make something whose type is `null | undefined | value` without a specific reason to do so. It's weird that `null !== undefined` are two different things in JavaScript, but given that they are, one should always be careful about not mixing the types -- it can lead to some pretty strange bugs.

Reply:
I understand, and I'll remember to be extra careful about this moving forward. Thanks as always for the knowledge! :)